app.go

```diff
@@ -261,6 +261,10 @@ type Static struct {
 	// The name of the index file for serving a directory.
 	// Optional. Default value "index.html".
 	Index string `json:"index"`
+
+	// The value for the Cache-Control that is set on the file response.
```

Comment:
You might want to clarify that the `int` argument defines as `seconds`

Reply:
Good catch! I've added a better explanation. Thinking out loud: What about an external link to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control?